src/Components/CustomModal.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Modal from 'react-modal';
+import '../styles/Modal.css';
+import '../styles/Button.css';
+
+export default function CustomModal({
+  modalIsOpen,
+  modalLabel,
+  modalMessage,
+  closeFunction,
+  confirmFunction,
+}) {
+  return (
+    <Modal
+      isOpen={modalIsOpen}
+      onRequestClose={closeFunction}
+      contentLabel={modalLabel || 'Alert'}
+      className="Modal"
+      overlayClassName="ModalOverlay"
+    >
+      <p>{modalMessage}</p>
+      <div className="ModalButtons">
+        <button className="Button cancelDelete" onClick={closeFunction}>
+          {confirmFunction ? 'No' : 'Close'}
+        </button>
+
+        {confirmFunction && (
```

Comment:
Similar to the comment above, I really like how you've extended the functionality of the modal to conditionally render a "Yes" button if necessary